renderer/svg.go

```diff
@@ -25,34 +25,58 @@ func UsePNGConverter(p g2s.PNGConverter) {
 	pngConverter = p
 }
 
+// valueAndColour represents a choropleth data point, which has both a numeric value and an associated colour
 type valueAndColour struct {
 	value  float64
 	colour string
 }
 
-// RenderSVG generates an SVG map for the given request
-func RenderSVG(request *models.RenderRequest) string {
+// SVGRequest wraps a models.RenderRequest and allows caching of expensive calculations (such as converting topojson to geojson)
```

Comment:
are these things cached per-request, or across requests? if per-request, is there any value in caching them more globally/for longer?

Reply:
The values being cached are specific to the request, sadly there's nothing we can cache between requests.